lib/registries/addon/drafts/draft/navigation-manager.ts

```diff
@@ -52,7 +52,7 @@ export default class NavigationManager {
         return pageManagers.length - 1;
     }
 
-    @computed('currentPage', 'pageManagers.[]')
+    @computed('currentPage', 'pageManagers.[]', 'currentRoute')
```

Comment:
hm, seems like you're applying the `@tracked` mental model here (which is great! keep using that) but computed properties should list the dependencies they *directly* use (in this case `pageManagers`, `currentPage`, `lastPage`, and `inMetadata`), not the state that they *indirectly* use (`currentRoute`) -- the dependency graph leading to `currentRoute` is built automatically.

this fix works because `inMetadata` depends on `currentRoute`, but it's brittle because it's hard to verify at a glance whether the listed dependencies are correct, and hard to notice when things get out of sync (these are also reasons why `@tracked` is nicer)